HealthcareSystem/PatientWebApp/Templates/WelcomeTemplate.html

```diff
@@ -0,0 +1,169 @@
+
+                            <!-- END Header -->
+                            <!-- Intro -->
+                            <table width="100%" border="0" cellspacing="0" cellpadding="0">
+                                <tr>
+                                    <td style="padding-bottom: 10px;">
+                                        <table width="100%" border="0" cellspacing="0" cellpadding="0">
+                                            <tr>
+                                                <td class="tbrr p30-15" style=" padding: 60px 30px; border-radius: 26px 26px 0px 0px;" bgcolor="#12325c">
+                                                    <table width="100%" border="0" cellspacing="0" cellpadding="0">
+                                                        <tr>
+                                                            <td class="h1 pb25"
+                                                                style="color: #ffffff; font-family: 'Muli', Arial, sans-serif; font-size: 40px; line-height: 46px; text-align: center; padding-bottom: 25px;">
+                                                                Welcome to our Hospital, [username]
+                                                            </td>
+                                                        </tr>
+                                                        <tr>
+                                                            <td class="text-center pb25" style=" color: #c1cddc; font-family: 'Muli', Arial, sans-serif;font-size: 16px; line-height: 30px; text-align: center; padding-bottom: 25px; ">
+                                                                You are currently registered using
+                                                                <span class="m-hide"><br /></span>[email]
+                                                            </td>
+                                                        </tr>
+                                                        <!-- Button -->
+                                                        <tr>
+                                                            <td align="center">
+                                                                <table class="center" border="0" cellspacing="0" cellpadding="0" style="text-align: center;">
+                                                                    <tr>
+
+                                                                        <td class="pink-button text-button" style=" background: #006699;color: #c1cddc;font-family: 'Muli', Arial, sans-serif;font-size: 14px; line-height: 18px;padding: 12px 30px;text-align: center;border-radius: 0px 22px 22px 22px;font-weight: bold;">
+
+                                                                            <a href="http://localhost:65117/html/activate_account.html?id=[jmbg]" target="_blank" class="link-white" style="color: #ffffff;text-decoration: none;">Click here </a>
```

Comment:
This path definitely needs to be configurable, otherwise it's going to be an issue when it's time to deploy this. Maybe do something like "[host]/activate_account.html?id=[jmbg]" and either put the host in the config or somehow get it from ASP (I feel like that should be possible?). Or just do "[activation_path]" and assemble the whole path in the MailService.